presto-client/src/main/java/com/facebook/presto/client/ClientSession.java

```diff
@@ -134,6 +137,14 @@ public ClientSession(
             checkArgument(charsetEncoder.canEncode(entry.getKey()), "Credential name is not US_ASCII: %s", entry.getKey());
             checkArgument(charsetEncoder.canEncode(entry.getValue()), "Credential value is not US_ASCII: %s", entry.getValue());
         }
+
+        // verify the extra credentials are valid
```

Comment:
I think this is a copy paste from the prev section :) 
Please update the comments.